src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -49,14 +54,19 @@ void buildSearchConceptsQuery(CloudPlatform platform) {
   @ParameterizedTest
   @EnumSource(CloudPlatform.class)
   void buildSearchConceptsQueryEmpty(CloudPlatform platform) {
-    assertThat(
-        "generated SQL for empty search string is correct",
-        SearchConceptsQueryBuilder.buildSearchConceptsQuery(
-            "Condition", "", s -> s, CloudPlatformWrapper.of(platform)),
-        equalToCompressingWhiteSpace(
-            "SELECT c.concept_name, c.concept_id FROM concept AS c "
-                + "WHERE c.domain_id = 'Condition' "
-                + "LIMIT 100"));
+    CloudPlatformWrapper cloudPlatformWrapper = CloudPlatformWrapper.of(platform);
+    String actual = createQueryWithLimit().renderSQL(CloudPlatformWrapper.of(platform));
+    String expected = "SELECT t.* FROM table AS t";
+    if (cloudPlatformWrapper.isAzure()) {
```

Comment:
Doesn't this duplicate `QueryTest.renderSQLWithLimit()`?

Reply:
Thanks for pointing this out! It should be fixed.